Header header
float32 altitude
float32 accuracy
